core/cli/src/help.ts

```diff
@@ -1,13 +1,88 @@
 import { loadConfig } from './config'
 import { OptionKey, setOptions } from '@dotcom-tool-kit/options'
-import { styles } from '@dotcom-tool-kit/logger'
+import { styles as s } from '@dotcom-tool-kit/logger'
 import type { Logger } from 'winston'
+import YAML from 'yaml'
+import $t from 'endent'
+import { CommandTask, OptionsForTask } from '@dotcom-tool-kit/plugin'
+import { ValidConfig } from '@dotcom-tool-kit/config'
 
-export default async function showHelp(logger: Logger, hooks: string[]): Promise<void> {
+const toolKitIntro = s.box(
+  $t`
+    Tool Kit is modern, maintainable & modular developer tooling for FT.com projects.
+    ${s.URL('https://github.com/financial-times/dotcom-tool-kit')}
+  `,
+  { headerText: `🧰 ${s.title(`welcome to ${s.app('Tool Kit')}!`)}` }
+)
+
+const formatTask = ({ task, options }: OptionsForTask) => $t`
+  ${s.task(task)}${
+  Object.keys(options).length > 0
+    ? ` ${s.dim('with options:')}
+    ${YAML.stringify(options).trim()}`
+    : ''
+}`
+
+const formatCommandTasks = (config: ValidConfig, commands: string[]) =>
+  s.box(
+    $t`
+  ${s.help(
+    `${s.command('commands')} run Tool Kit tasks with ${s.code(
+      'npx dotcom-tool-kit $command'
+    )}, or via configuration installed by hooks in your repository.`
```

Comment:
**praise**: i think a lot of people will be helped by putting Tool Kit into context like this